static/js/settings.js

```diff
@@ -189,6 +189,11 @@ const setUpButtonsListeners = () => {
   const submitBtn = document.getElementById('submitBtn');
   const addExtraIDBtn = document.getElementById('add-extra-id');
   const addBlacklistedIDBtn = document.getElementById('add-blacklisted-id');
+  const changingInputs = [
```

Comment:
Move this below, right above where you use them, so it's logically closer. You could also just store the id and add the document.getElementById in the loop.